### PR TITLE
Improved docstrings of various transform methods on CanvasItem

### PR DESCRIPTION
**Pull request by fHachenberg**
The descriptions of various transform methods were incorrect or confusing.

### CODE REVIEW

File: doc/classes/CanvasItem.xml [hunk 1]
```diff
@@ -340,13 +340,13 @@
...
 			<description>
-				Returns the global transform matrix of this item.
+				Returns the global transform matrix of this item with respect to its canvas layer.
 			</description>
 		</method>
 		<method name="get_global_transform_with_canvas" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the global transform matrix of this item in relation to the canvas.
+				Returns the global transform matrix of this item with respect to the viewport. This is the same as get_canvas_transform() * get_global_transform(()
```
Comment (mhilbrunner) [inline doc/classes/CanvasItem.xml:349]:
```suggestion
				Returns the global transform matrix of this item with respect to the viewport. This is the same as [code]get_canvas_transform() * get_global_transform()[/code].
```

**Review by mhilbrunner — COMMENTED** [review]
Thanks for contributing! Some small nitpicks below. Also, please use a more specific commit message, like the title/description of this Pull Request, which is fine :)

File: doc/classes/CanvasItem.xml [hunk 2]
```diff
@@ -370,7 +370,7 @@
 		<method name="get_viewport_transform" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns this item's transform in relation to the viewport.
+				Returns this item's canvas layer's transformation with respect to the screen. Meaning: applying this to a Vector2 given relative to this item's canvas layer, it returns the screen coordinates of this Vector2.
```
Comment (KoBeWi) [inline doc/classes/CanvasItem.xml:373]: > screen

This is incorrect. Viewport and screen is not the same.

File: doc/classes/CanvasItem.xml (hunk 1 above)
Comment (KoBeWi) [inline doc/classes/CanvasItem.xml:349]: > viewport

This is inconsistent with how you changed canvas to canvas layer above. Is this intended?

Comment (Calinou) [reply doc/classes/CanvasItem.xml:349]: @fHachenberg Bump :slightly_smiling_face:

**Comment by YuriSizov** [comment]
Closing due to lack of response to feedback. If these improvements are still desired, they can be made with a new PR or by reviving this PR.

**Pull request closed without merging**